src/services/api/InterviewService.ts

```diff
@@ -0,0 +1,18 @@
+import apiClient from '@services/api/axios';
+import { ApiResponse } from '@type/api';
+import { InterviewUuid } from '@type/interview';
+
+export interface InterviewCreateData {
+  categoryIdList: number[];
+}
+
+export interface InterviewUuidData {
+  interviewUuid: InterviewUuid;
+}
```

Comment:
P4: 인터뷰 생성하면서 오는 정보를 묶는 의미에서 InterviewInfo같이 조금 포괄적인 정보를 의미하는 네이밍으로 변경해봐도 괜찮을 것 같습니다!
```suggestion
export interface InterviewInfo {
  interviewUuid: InterviewUuid;
}
```

Reply:
해당 데이터는 인터뷰 정보가 아니라 진짜 UUID만 받을 예정이라 저렇게 지었습니다!
다음 페이지로 이동하면서 인터뷰 정보 받는 데이터를 `InterviewInfoData`로 지어보겠습니다!

Reply:
InterviewUuid 여기서 `UUID`라는 부분을 지워보면 어떨까요?  
uuid는 식별자를 생성하는 방법 중에 하나라 어떻게 보면 대체될 수 있는 부분이여서 필드 네이밍에서는 uuid 부분이 빠져도 되지 않을까하는 생각이 있습니다!

ex) jwt token 부분 응답으로 주는 부분에서 jwtToken 이라고 묶지 않고 그냥 token으로 리소스 묶어서 주는 것 처럼요!

Reply:
좋은 의견 같습니다! `code` 로 변경해서 별도 태스크로 리팩토링 진행해보도록 하겠습니다.